### PR TITLE
Fix typo and change to match statement

### PR DESCRIPTION
**Pull request by NoCtrlZ**
There are two changes.
- I found a typo so I fixed it
- I changed the `if` statement to the `match` statement

### CODE REVIEW

File: src/commitment_scheme/kzg10/key.rs [hunk 1]
```diff
@@ -116,29 +116,29 @@ impl CommitKey {
...
+            // Check that the truncated degree is not zero
+            0 => Err(Error::TruncatedDegreeIsZero),
+            // Check that max degree is less than truncated degree
+            i if i > self.max_degree() => Err(Error::TruncatedDegreeTooLarge),
+            _ => {
+                let truncated_powers = Self {
+                    powers_of_g: self.powers_of_g[..=truncated_degree].to_vec(),
+                };
+                Ok(truncated_powers)
+            }
```
Comment (CPerezz) [inline src/commitment_scheme/kzg10/key.rs:140]: That looks great! Although we could reduce a bit the code and unify two arms of the match statement.

What do you think about this?:

```suggestion
      // Check that the truncated degree is not zero
      0 => Err(Error::TruncatedDegreeIsZero),
      // Check that max degree is less than truncated degree
      i if i > self.max_degree() => Err(Error::TruncatedDegreeTooLarge),
      i => {
              if i == 1 {truncated_degree += 1};
              let truncated_powers = Self {
              powers_of_g: self.powers_of_g[..=truncated_degree].to_vec(),
    };
    Ok(truncated_powers)
}      
```

**Review by CPerezz — COMMENTED** [review]
LGTM. Maybe worth considering the purposed changes. WDYT? @NoCtrlZ

File: src/commitment_scheme/kzg10/key.rs (hunk 1 above)
Comment (NoCtrlZ) [reply src/commitment_scheme/kzg10/key.rs:140]: Amazing!
I'd have liked to avoid the duplicate but I didn't know this one...
Really cool.
Thank you.

Comment (NoCtrlZ) [reply src/commitment_scheme/kzg10/key.rs:140]: I changed!
Please confirm.

**Comment by NoCtrlZ** [comment]
I deleted my fork repository mistakingly so let me move to another [PR](https://github.com/dusk-network/plonk/pull/545).

**Pull request closed without merging**